cosa/analyzers/bmc.py

```diff
@@ -778,6 +805,59 @@ def _remap_model(self, vars, model, k):
         Logger.error("Invalid configuration strategy")
         return None
 
+    def sim_no_unroll(self, hts, cover, k):
```

Comment:
I would make it incremental:
add_assertion(trans 0 1)

for i < k:
push()
add_assertion(init)
solve()
init = get_model(t=0)
pop()